etcdmain/config.go

```diff
@@ -131,6 +131,7 @@ func newConfig() *config {
 	fs.Var(cfg.ec.CorsInfo, "cors", "Comma-separated white list of origins for CORS (cross-origin resource sharing).")
 	fs.StringVar(&cfg.ec.Dir, "data-dir", cfg.ec.Dir, "Path to the data directory.")
 	fs.StringVar(&cfg.ec.WalDir, "wal-dir", cfg.ec.WalDir, "Path to the dedicated wal directory.")
+	fs.StringVar(&cfg.ec.LocalAddr, "local-address", cfg.ec.LocalAddr, "Local adddress bind for outgoing client connections.")
```

Comment:
we need to mention this is for peer communication as other flags. probably also add a peer prefix.

Reply:
What do you suggest? peer-local-address?

Reply:
yea.